submit.go

```diff
@@ -470,70 +471,69 @@ func (s *submitServer) saveReport(ctx context.Context, p parsedPayload, reportDi
 		return nil, err
 	}
 
+	if err := s.sendEmail(p, reportDir); err != nil {
+		return nil, err
+	}
+
 	return &resp, nil
 }
 
 func (s *submitServer) submitGithubIssue(ctx context.Context, p parsedPayload, listingURL string, resp *submitResponse) error {
-	if s.ghClient == nil {
```

Comment:
I'm failing to see where we now handle the cases of github/slack/email being disabled? we seem to call `submitGithubIssue` unconditionally so won't it just fail?

Reply:
... Indeed. Fixing that.